package.json

```diff
@@ -53,30 +53,34 @@
     ]
   },
   "dependencies": {
-    "@react-native-community/async-storage": "^1.5.0",
+    "@react-native-community/async-storage": "^1.6.2",
     "@react-navigation/core": "^3.5.1",
     "bugsnag-react-native": "2.23.2",
+    "lodash": "4.17.15",
```

Comment:
Not sure why introduced, is it a peer dependancy for something?

Reply:
Mustv'e slipped through. I didn't add it explictly so mustv'e been a peer dependency. I'll try and look or just revert?

Reply:
I don't think it can get in there by itself? Just quickly remove and check if it whines?

Reply:
41aa639